lib/iris/__init__.py

```diff
@@ -108,7 +108,7 @@ def callback(cube, field, filename):
 
 
 # Iris revision.
-__version__ = '1.7.4-DEV'
+__version__ = '1.7.4rc1'
```

Comment:
Really? I'd be comfortable for going straight to release. If there is something wrong with it, we've clearly misunderstood what a patch release means... and would therefore revert back to v1.7.3.

No comment necessary, just a point for future decisioning re versioning.